### DISCUSSION TITLE
Can't "make" it. What do I wrong?

### DISCUSSION THREAD

**Discussion by Moktur** [KinesisCorporation/Adv360-Pro-ZMK#539] [Q&A]
Good day to you, fellow users of the adv360zmk.

I forked and customized my own keymap, but somehow I am not able to compile it. I receive this strange error. Don't know really what to do, because I have not much experience with Docker and Make.

![image](https://github.com/user-attachments/assets/21e64529-f485-4ce2-af9a-2e850ee0b47a)

I am thanksfull for any help.

Kind regards,

pepe

**Comment by ReFil** [comment]
Hi, you should be able to use the online GitHub building, it is a lot easier than the local build process
